azure-communication-ui/calling/src/main/java/com/azure/android/communication/ui/calling/service/CallingService.kt

```diff
@@ -3,18 +3,20 @@
 
 package com.azure.android.communication.ui.calling.service
 
+/*  <CALL_START_TIME> */
```

Comment:
Broken flag for call start time. (covers all imports).

Recommend removing, as if the feature gets disabled, the error will propagate anyways, and can then be fixed in the main class file (i.e. by just stubbing out it's impl, but allowing the import to remain without flags).